launcher/container_runner.go

```diff
@@ -236,14 +266,38 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 		return nil, fmt.Errorf("failed to create REST verifier client: %v", err)
 	}
 
+	// Fetch container image signatures by using signaturediscovery client.
```

Comment:
If you don't update the spec, this func should be unnecessary. Which is good because this is tough to follow.